docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+  roopapi:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    volumes:
+      - roop-data:/roop
+      - open-nsfw:/root/.opennsfw2
```

Comment:
can you explain why that is exposed? all dependencies should be inside the container

Reply:
The idea was to save the models in the volume. So if the container crashes, or the host pc shutsdown, the models remain and a new download is not required.